engine.go

```diff
@@ -99,7 +99,7 @@ func (w workflowEngine) Parse(
 		return nil, err
 	}
 	if stepWorkflowFileCache != nil {
-		files = loadfile.MergeFileCaches(stepWorkflowFileCache, files)
+		files, _ = loadfile.MergeFileCaches(stepWorkflowFileCache, files)
 	}
```

Comment:
After going to the trouble of modifying `MergeFileCaches()` to return it, deliberately ignoring the error is an interesting choice.  Is anyone going to notice that the error was logged if we carry on despite it, here?